DMS Viewer/DMSParser.cs

```diff
@@ -19,7 +19,7 @@ public class DMSParser
         Regex startedRegex = new Regex(@"REM Started: (.*)");
 
         Regex tableNameRegex = new Regex(@"EXPORT\s+(.*?)\.(.*?)\s+(WHERE)?");
-        Regex columnRegex = new Regex(@"(([A-Z0-9_]+):(2?[A-Z]+)\((\d+)\)~~~).*?");
+        Regex columnRegex = new Regex(@"(([A-Z0-9_]+):(\d?[A-Z]+)\((\d+)(,?\d*?)\)~~~).*?");
```

Comment:
I think the (,?\d*?) portion can be written as (,\d+)? since we'll never have 2 numbers without a comma, and never a comma without a 2nd number.